src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,187 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int MULTIPLIER_FOR_INCREASE_CAPACITY = 2;
+    private Node<K, V>[] table;
+    private int size;
+    private int threshold;
+    private int capacity;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_INITIAL_CAPACITY];
```

Comment:
Use the diamond operator to ensure type safety when instantiating the Node array: new Node<K,V>[DEFAULT_INITIAL_CAPACITY].